spec/unit/installer/file_references_installer_spec.rb

```diff
@@ -40,13 +40,38 @@ module Pod
         file_ref.path.should == 'libBananalib.a'
       end
 
-      it 'adds the files references of the resources the Pods project' do
+      it 'adds files references for the resources of the Pods project' do
         @installer.install!
         file_ref = @installer.pods_project['Pods/BananaLib/Resources/logo-sidebar.png']
         file_ref.should.be.not.nil
         file_ref.path.should == 'Resources/logo-sidebar.png'
       end
 
+      it "add file references for localization directories if glob doesn't include contained files" do
+        @installer.install!
+        file_ref = @installer.pods_project['Pods/BananaLib/Resources/en.lproj']
+        file_ref.should.be.not.nil
+      end
+
+      it 'adds file references for files within CoreData directories' do
+        @installer.install!
+        model_ref = @installer.pods_project['Pods/BananaLib/Resources/Sample.xcdatamodeld']
+        model_ref.should.be.not.nil
+
+        # Files within the .xcdatamodeld directory are added automatically by adding the .xcdatamodeld directory.
+        file_ref = @installer.pods_project['Pods/BananaLib/Resources/Sample.xcdatamodeld/Sample.xcdatamodel']
+        file_ref.should.be.not.nil
+      end
+
+      it "doesn't add file references for files within Asset Catalogs" do
```

Comment:
Shouldn't this be inside `'Installation With Recursive Resources Glob'` section? BananaLib adds resources non-recursively by default, also explicitly sets xcassets path. This scenario seems to not testing your aim because there will be no resource paths for Pods/BananaLib/Resources/Images.xcassets/Logo.imageset/logo.png for example. Am I missing sth?